persistence/schemas/de.cyface.persistence.DatabaseV6/1.json

```diff
@@ -0,0 +1,120 @@
+{
+  "formatVersion": 1,
+  "database": {
+    "version": 1,
+    "identityHash": "c3ec4780488bc825ddc6571e20a59607",
+    "entities": [
+      {
+        "tableName": "Pressure",
+        "createSql": "CREATE TABLE IF NOT EXISTS `${TABLE_NAME}` (`pressure` REAL NOT NULL, `measurement_fk` INTEGER NOT NULL, `uid` INTEGER PRIMARY KEY AUTOINCREMENT NOT NULL, `timestamp` INTEGER NOT NULL)",
```

Comment:
In order for `measurement_fk` to be `NOT NULL` as expected in the database I was not able to make the measurement id optional, as I did before.

For that reason I added a separate class for a persisted `Pressure` and `GeoLocationV6`.

This also cleans up the code as
- we only implement parcelable for the cached classes `Pressure` and `GeoLocationV6` but not for the persisted classes as we never send them though parcels
- we ensure that pressure data and locations cannot be inserted into the database without a linked measurement, as expected.